spk/firebird/src/wizard/install_uifile

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "step_title": "Configuration of FirebirdSQL server",
+    "items": [
+      {
+        "type": "password",
+        "desc": "SYSDBA user",
```

Comment:
Is sysdba the same as a root account? Maybe reword it slightly for users. You can keep sysdba but maybe add a more detailed description.

Reply:
Yes, it is the database superuser in the [Firebird parlance](https://firebirdsql.org/file/documentation/html/en/firebirddocs/qsg3/firebird-3-quickstartguide.html#qsg3-config). I believe that anyone who will work with Firebird will be aware of this.
Adding a little explanation can't hurt for sure :)

Reply:
Done. I extended the install wizard description.